script/state.py

```diff
@@ -13,6 +15,10 @@
 states_data = cur.fetchall()
 conn.close()
 
+if len(states_data)==0:
```

Comment:
Although this way is correct and will work, just for information, we can also write:
```
if not states_data
```
or 
```
if not len(states_data)
```
all will work

Reply:
Done